crates/apub/src/fetcher/mod.rs

```diff
@@ -58,7 +58,7 @@ where
     let identifier = identifier.to_string();
     Ok(
       blocking(context.pool(), move |conn| {
-        DbActor::read_from_name(conn, &identifier, false)
+        DbActor::read_from_name(conn, &identifier, true)
```

Comment:
You are setting the param `include_deleted` to true, i dont think thats what you want.

Reply:
Yep... when I was testing this, I couldn't even view the removed community's page as a mod to unremove it. The urls should still be accessible, just not visible from the main feeds.

Reply:
This code is used when a local user/community is searched with `@user` or `!community`. With the existing code, deleted/removed items are not shown. So you should revert this change, unless you noticed a bug in the code.

Reply:
Its also used by api_crud/community/read.rs, which is why after removing a community, it was incorrectly giving 404s:

https://github.com/LemmyNet/lemmy/blob/main/crates/api_crud/src/community/read.rs#L46

Reply:
Hmm, i guess you need to add a param `include_deleted` to fn resolve_actor_identifier(). Because at least for search that param should be false, not sure about other uses.

Reply:
I'll add this shortly.